README.rst

```diff
@@ -7,16 +7,16 @@ OpenSCM
 | |Codecov|      | |Codacy| |
 +----------------+----------+
 
+.. sec-begin-long-description
 .. sec-begin-index
 
 The **Open Simple Climate Model framework** unifies access to several
 simple climate models (SCMs). It defines a standard interface for
 getting and setting model parameters, input and output data as well as
 for running the models. Additionally, OpenSCM provides a standardized
 file format for these parameters and scenarios including functions for
-reading and writing such files. Its high-level interface further adds
-convenience functions and easily enables stochastic ensemble runs,
-e.g. for model tuning.
+reading and writing such files. It further adds convenience functions
+and easily enables stochastic ensemble runs, e.g. for model tuning.
```

Comment:
```suggestion
and easily enables ensemble runs, e.g. for scenario assessment or model tuning.
```

Reply:
Just as an fyi, given the work elsewhere, I think putting model calibration into OpenSCM would be a mistake (it's a separate application with its own pitfalls and tricks). OpenSCM should be able to run an ensemble of parameters generated from a calibration library and analyse the output, but it should not have a function like `model.tune_to_target(target)`.

Reply:
a pity, but ok

Reply:
To clarify, we're still going to do it. It's just a separate library (putting it in one library makes it way too big). Hopefully that means it's not such a pity?